lib/cfg-parser.c

```diff
@@ -121,8 +121,9 @@ static CfgLexerKeyword main_keywords[] =
   { "time_sleep",         KW_TIME_SLEEP, KWS_OBSOLETE, "time_sleep() has been deprecated" },
   { "file_template",      KW_FILE_TEMPLATE },
   { "proto_template",     KW_PROTO_TEMPLATE },
-  { "default_level",      KW_DEFAULT_LEVEL },
-  { "default_priority",   KW_DEFAULT_LEVEL },
+  { "default_level",      KW_DEFAULT_SEVERITY },
```

Comment:
shouldn't we use `KWS_OBSOLETE` to indicate that we intend to deprecate it?

Reply:
My understanding about the final conclusion was that we are not actively deprecating the aliases, as that would push a lot of work to anyone using these options and maintaining the alias is not that much of a maintenance nightmare. the level() filter is pretty common in any configuration. And I would deprecate these options as a whole (e.g. level(), default-level(), $LEVEL at the same time).